src/main/webapp/components/directives/cache/security/security.js

```diff
@@ -0,0 +1,29 @@
+(function () {
```

Comment:
I see that there is separate directive for each cache, that is because they are so different? It seems like a good approach, just asking!